frontend/test/lit/test_peephole_optimizations.py

```diff
@@ -46,35 +46,6 @@ def flush_peephole_opted_mlir_to_iostream(QJIT):
     shutil.rmtree(QJIT.__name__)
 
 
-#
-# General lowering tests
-#
-
-
-def test_transform_named_sequence_injection():
```

Comment:
Nice! I never liked the idea of having the (possibly empty) transform sequence on every mlir we produce (although I was the one who added it...)

Reply:
Well, there is sitll the possibility of an empty transform sequence and now one in each submodule. But we can possibly also remove that in a following PR.